docs/organizations/managing-security-and-risk.md

```diff
@@ -27,6 +27,8 @@ The main area of the dashboard includes five panels:
 
 Each panel shows the total count of matching items and contains a **Review** button to view a list of matching items.
 
+To limit the total counts in each panel to a specific set of severities or repositories, click the **Severity** or **Repository** drop-downs above the main area.
```

Comment:
```suggestion
To limit the total counts in each panel to a specific set of severities or repositories, click the **Severity** or **Repository** filters above the main area.
```
same as the comment below, wdyt?

Reply:
We'll leave this as-is, see [previous comment](https://github.com/codacy/docs/pull/1797#discussion_r1270442680).